### ISSUE TITLE
Request blog entries on hacks

### ISSUE THREAD

**Issue by kbarbary** [AstroHackWeek/AstroHackWeek2015#22]
Should we send out an email with info telling people how to submit blog entries? I can manage the Nikola blog site.

If so, we should also create a schedule of when the entries will go live.

**Comment by dhuppenkothen** [comment]
Yes, it's a good idea! Depending on responses, perhaps, we might want to request blog entries from specific people.

Think we can manage to put something live every week?

**Comment by kbarbary** [comment]
Depends on how much we nag people. :)
